project/plugins.sbt

```diff
@@ -21,3 +21,17 @@ addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.17")
 addSbtPlugin("org.musigma" % "sbt-rat" % "0.7.0")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.3")
 addSbtPlugin("com.github.battermann" % "sbt-json" % "0.5.0")
+
+/**
+  * These dependencies are used in a effort to support checking dependencies
+  * for CVEs and making sure they are up to date. As well being able to track
+  * dependencies of dependencies.
+  *
+  * Commands:
+  *   Check dependencies for CVEs -> sbt dependencyCheckAggregate
+  *   Print out dependency tree -> sbt dependencyTree
+  *   Check for available dependency updates -> sbt dependencyUpdates
+  */
+addDependencyTreePlugin
+addSbtPlugin("net.vonbuchholtz" % "sbt-dependency-check" % "4.0.0")
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.1")
```

Comment:
Personally, I think these plugins should be part of ~/.sbt/plugins/plugins.sbt rather than the repo since these aren't strictly needed for the vscode repo. Minimizes dependencies for people that don't care about this this stuff. But I don't feel that strongly about it.